fixups/RegLegacyFixups/InitializeFixup.cpp

```diff
@@ -117,12 +121,102 @@ void LogString(const wchar_t* name, const wchar_t* value)
     Log(L"%ls=%ls\n", name, value);
 }
 
+struct EntryToCreate 
+{
+    std::wstring path;
+    std::unordered_map<std::wstring, std::wstring> values;
+};
+
 
-void InitializeFixups()
+void CreateRegistryRedirectEntries(std::wstring_view dependency_name, std::vector<EntryToCreate> entries)
 {
+    Log("RegLegacyFixups Create redirected registry values\n");
+
+    Package pkg = Package::Current();
+    IVectorView<Package> dependencies = pkg.Dependencies();
+
+    std::wregex dependency_version_regex = std::wregex(L"%dependency_version%");
+    std::wregex dependency_path_regex = std::wregex(L"%dependency_root_path%");
+
+    Package dependency = nullptr;
+
+#ifdef _DEBUG
```

Comment:
Please consolodate all `#if`'s to one method.

Reply:
I am conditionally compiling log printing. How do you consolidate it?

Reply:
Anyway I removed them. Log implementation is anyway behind _DEBUG

Reply:
> I am conditionally compiling log printing. How do you consolidate it?

My suggestion would be to always call log and in `log` use `#if`